planner/core/memtable_predicate_extractor.go

```diff
@@ -533,7 +533,22 @@ func (e *ClusterTableExtractor) Extract(_ sessionctx.Context,
 }
 
 func (e *ClusterTableExtractor) explainInfo(p *PhysicalMemTable) string {
-	return ""
+	if e.SkipRequest {
+		return "skip_request: true"
```

Comment:
```suggestion
		return "skip_request:true"
```

to be consistent with other places.